### PR TITLE
Kill shells

### PR DESCRIPTION
**Pull request by ekohl**
I should note that for me it's not rsyncing /vagrant (though it says it does) meaning it doesn't actually work for me. That makes me wonder if we should just convert this last shell to ansible.

### CODE REVIEW

File: config/base_boxes.yaml [hunk 1]
```diff
@@ -50,13 +46,7 @@ boxes:
         group: 'freeipa_server'
         server: 'centos7-katello-nightly'
 
-  centos7-bats:
-    box: centos7
-    ansible:
-      playbook: 'playbooks/bats_pipeline_nightly.yml'
-      group: 'bats'
-
```
Comment (beav) [inline config/base_boxes.yaml]: is this removal intentional? I'm not sure how it is related to the shells removal

Comment (ekohl) [reply config/base_boxes.yaml]: It can be removed but you're right that it doesn't belong in this PR.

**Comment by beav** [comment]
ACK from me, thanks for getting rid of this extra piece of abstraction.

**Comment by ekohl** [comment]
Rebased to re-add the bats box. I'll have a look at cleaning up the base boxes after https://github.com/theforeman/forklift/pull/393 is merged.

**Comment by beav** [comment]
ACK again!
